.drone.yml

```diff
@@ -0,0 +1,72 @@
+pipeline:
+
+# staging
+
+  build_stg:
+    image: alpine:latest
+    commands:
+      - apk add --update make bash curl
+      - export BASE_URL="//blog.srcd.run/"
+      - make build
+    when:
+      branch: master
+
+  docker_stg:
+    image: plugins/docker
+    registry: quay.io
+    repo: quay.io/srcd/blog
+    secrets: [ docker_username, docker_password ]
+    tag: ${DRONE_COMMIT_SHA:0:7}
+    file: Dockerfile
+    when:
+      branch: master
+
+  helm_deploy_stg:
+    image: quay.io/ipedrazas/drone-helm
+    skip_tls_verify: true
+    chart: ./helm-charts/blog
+    release: blog
+    prefix: STG
+    secrets: [ STG_API_SERVER, STG_KUBERNETES_TOKEN ]
+    values: ingress.globalStaticIpName=blog-srcd-run,ingress.hostname=blog.srcd.run,image.repository=quay.io/srcd/blog,image.tag=${DRONE_COMMIT_SHA:0:7}
+    tiller_ns: kube-system
+    debug: true
+    wait: true
+    when:
+      branch: master
+
+
+# production
+
+  build_prod:
+    image: alpine:latest
+    commands:
+      - apk add --update make bash curl
+      - export BASE_URL="//blog.sourced.tech/"
+      - make build
+    when:
+      event: tag
+
+  docker_prod:
+    image: plugins/docker
+    registry: quay.io
+    repo: quay.io/srcd/blog
+    secrets: [ docker_username, docker_password ]
+    tag: ${DRONE_TAG}
+    file: Dockerfile
+    when:
+      event: tag
+
+  helm_deploy_prod:
+    image: quay.io/ipedrazas/drone-helm
```

Comment:
I'm not sure that we want ipedrazas image... I'd rather fork his repo and create one for us as I don't know how stable that image is going to be.

Reply:
Well, it's the only helm plugin that is linked in [Drone's website](http://plugins.drone.io/), and the one that the community is using and contributing to. I'd say it's stable enough.

Reply:
Fair enough